LinqOverWCF/Windows Host/Client/LinqToDB.Templates/CopyMe.Access.tt.txt

```diff
@@ -0,0 +1,54 @@
+<#@ template language="C#" debug="True" hostSpecific="True" #>
```

Comment:
maybe remove template files from repo as they should be restored from nuget anyway?
BTW, migration to CPS also will fix it as CPS project will use files from nuget cache instead of local copy